tools/ClassGen/NodeGen.cpp

```diff
@@ -105,6 +105,8 @@ int main(int argc, char **argv) {
       .addInput("Input")
       .addInput("Weights")
       .addInput("Bias")
+      .addMember(MemberType::SizeT, "Axis")
+      .addMember(MemberType::SizeT, "AxisW")
```

Comment:
Why do we need this AxisW? As far as I understand, it is always =1, since you do flattening in Caffe2 Importer.

Reply:
Yes, in Caffe2 importer, "weight" is flatten. But I think we should also support the param for FC in C++ ?